recipes/allennlp/meta.yaml

```diff
@@ -0,0 +1,126 @@
+{% set name = "allennlp" %}
+{% set version = "0.4.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dbc0b7de268a14ad978ad29ceb785c0a22cab13e2786f04fe933b4c71e9ad7aa" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  # Fix this when the new release is officially out.
+  url: https://github.com/allenai/allennlp/archive/v{{ version }}.tar.gz
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True  # [win or py>=36]
```

Comment:
Why did you change this to `py>=36`? It was correct when it was `py<36`

Reply:
Ah, seeing as it is 3.6 only I figured they’d be more or less the same. I will change it back, thanks.

Reply:
They are not the same. They are opposites of each other.

`py<36` skips python 2.7, 3.5 and runs 3.6
`py>=36` skips python 3.6 and runs 2.7, 3.5

Reply:
Oh whoops, you’re right, thanks for pointing that out. That probably explains the issue I’m having with ci